integration/init_test.go

```diff
@@ -39,12 +39,20 @@ func TestInit(t *testing.T) {
 			name: "getting-started",
 			dir:  "testdata/init/hello",
 		},
+		{
+			name: "microservices (backwards compatibility)",
+			dir:  "testdata/init/microservices",
+			args: []string{
+				"-a", `leeroy-app/Dockerfile=gcr.io/k8s-skaffold/leeroy-app`,
+				"-a", `leeroy-web/Dockerfile=gcr.io/k8s-skaffold/leeroy-web`,
+			},
+		},
 		{
 			name: "microservices",
 			dir:  "testdata/init/microservices",
 			args: []string{
-				"-a", "leeroy-app/Dockerfile=gcr.io/k8s-skaffold/leeroy-app",
-				"-a", "leeroy-web/Dockerfile=gcr.io/k8s-skaffold/leeroy-web",
+				"-a", `{"name":"Docker","payload":{"path":"leeroy-app/Dockerfile"}}=gcr.io/k8s-skaffold/leeroy-app`,
+				"-a", `{"name":"Docker","payload":{"path":"leeroy-web/Dockerfile"}}=gcr.io/k8s-skaffold/leeroy-web`,
 			},
 		},
```

Comment:
Is it possible to add Jib test here too?

Reply:
Maybe in a follow-up? This might require adding another test project just for this and I'd like to keep this PR small. The unit tests already cover parsing Jib payloads, at least.